controllers/operands/kubevirt.go

```diff
@@ -426,6 +431,14 @@ func getKVConfig(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.KubeVirtConfigu
 	if val, ok := os.LookupEnv(machineTypeEnvName); ok {
 		if val = strings.TrimSpace(val); val != "" {
 			config.MachineType = val
+
+			config.ArchitectureConfiguration = &kubevirtcorev1.ArchConfiguration{
+				Amd64: &kubevirtcorev1.ArchSpecificConfiguration{
+					MachineType:      val,
+					OVMFPath:         DefaultAMD64OVMFPath,
+					EmulatedMachines: strings.Split(DefaultAMD64EmulatedMachines, ","),
```

Comment:
Currently `MachineType` takes the value of  `MACHINETYPE` env variable that can be set at bundle build time.
Are `OVMFPath` and `EmulatedMachines` completely independent from `MachineType` or we should also introduce 2 additional optional env variables for them?

Reply:
They are 3 independent settings, we can, of course, give the option to set all 3 with env vars but I thought that for the sake of simplicity to just hardcode the remaining 2 to KubeVirt's default.